core/src/main/java/com/sequenceiq/cloudbreak/controller/v4/CredentialV4Controller.java

```diff
@@ -74,10 +63,10 @@ public CredentialV4Response get(Long workspaceId, String name) {
 
     @Override
     public CredentialV4Response post(Long workspaceId, CredentialV4Request request) {
-        User user = userService.getOrCreate(restRequestThreadLocalService.getCloudbreakUser());
+        // TODO
```

Comment:
![7a30ac17ae53ef3d2706432a7c772a26](https://user-images.githubusercontent.com/16577123/51118090-cd553400-180f-11e9-9ddd-a20b4997ff10.jpg)

Reply:
![Im watching you](https://i.kym-cdn.com/entries/icons/facebook/000/007/195/im_watching_you_-_copia.jpg)